src/wasm-interpreter.h

```diff
@@ -1105,7 +1114,20 @@ class ExpressionRunner : public OverriddenVisitor<SubType, Flow> {
     Literal c = flow.getSingleValue();
     switch (curr->op) {
       case Bitselect:
+      case LaneselectI8x16:
+      case LaneselectI16x8:
+      case LaneselectI32x4:
+      case LaneselectI64x2:
         return c.bitselectV128(a, b);
+
+      case RelaxedFmaVecF32x4:
+        return a.addF32x4(b.mulF32x4(c));
+      case RelaxedFmsVecF32x4:
+        return a.subF32x4(b.mulF32x4(c));
+      case RelaxedFmaVecF64x2:
+        return a.addF64x2(b.mulF64x2(c));
+      case RelaxedFmsVecF64x2:
+        return a.subF64x2(b.mulF64x2(c));
```

Comment:
I'm not interested, isn't it better to use soft fma / fms without rounding error for the interpreter? The interpreter is also used for constant folding. Another discussion about this: https://github.com/WebAssembly/binaryen/issues/3203#issuecomment-706392532

Reply:
Yea, we can `#include <math.h>` and use [fma](https://en.cppreference.com/w/c/numeric/math/fma). Non-fused implementation is also correct for relaxed instruction. So either is fine.

Reply:
Yeah, but it still required soft fallback if `FP_FAST_FMA` is not supported.
https://github.com/WebAssembly/design/issues/1391#issue-766900584

Reply:
What you linked is not the same idea as the relaxed-simd proposal's FMA instruction. A relaxed fma does not require a precise soft fallback. An implementation can either perform fused, or non-precise non-fused.

Reply:
What I mean is that binaryen's interpreter should be always be deterministic use most accurate version of fma / fms in any scenario. Since it is clear that it will be non-deterministic "by execution" and that's basic idea for relaxed simd. But as for me it should be deterministic "by compilation".

Reply:
Got it, make sense, fixed in latest commit